modules/local/nf-core-modified/quast/main.nf

```diff
@@ -0,0 +1,37 @@
+process QUAST {
```

Comment:
just curious, what modification did you do here?

Reply:
modifies how the output directory is named and copying a specific report.tsv file. Have added this description in comments